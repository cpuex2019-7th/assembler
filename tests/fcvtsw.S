    li t1, 100

    fcvtsw ft1, t1
    fmvxw a0, ft1
