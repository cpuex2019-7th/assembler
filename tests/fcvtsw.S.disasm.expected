addi t1, zero, 100
fcvtsw ft1, t1
fmvxw a0, ft1
